### PR TITLE
Add actions back to the swap page

### PR DESCRIPTION
**Pull request by luckysori**
It looks the way it did before:

- Communication actions:
Jun08::104321

- Ledger actions:
Jun08::104358

### CODE REVIEW

**Comment by da-kami** [comment]
❓ Could you maybe provide a screenshot how it looks?

**Comment by da-kami** [comment]
🙃  I think it would be more intuitive if the list of recorded events and actions would just go one list and not be split up in Alpha and Beta for "Ledger events and actions". I think it is difficult to understand for the user what he has to do. If there is just a simple list, where the last element either instructs the user to do an action (swap or ledger action) OR tells the user to wait for the other party to do something it would be easier to understand.

File: src/pages/SwapPage/Swap.tsx [hunk 1]
```diff
@@ -38,6 +58,67 @@ function Swap({ swap }: SwapProps) {
...
     properties.role === Role.Alice ? tradeActions : tradeActions.reverse();
 
+  const communicationActions = (swap.actions || []).filter(
+    action => action.name === "accept" || action.name === "decline"
+  );
+
+  const ledgerActions = (swap.actions || []).filter(
+    action => action.name !== "accept" && action.name !== "decline"
+  );
+  const alphaLedgerActions =
```
Comment (da-kami) [inline src/pages/SwapPage/Swap.tsx]: 📣 There seems to be some logic here that could be tested. Please add some UI tests to the component that verifies the workflow for the different roles.

Comment (luckysori) [reply src/pages/SwapPage/Swap.tsx]: I agree. I think in general the swap page component needs UI tests. I've opened #92 to track this.

**Comment by da-kami** [comment]
> 🙃 I think it would be more intuitive if the list of recorded events and actions would just go one list and not be split up in Alpha and Beta for "Ledger events and actions". I think it is difficult to understand for the user what he has to do. If there is just a simple list, where the last element either instructs the user to do an action (swap or ledger action) OR tells the user to wait for the other party to do something it would be easier to understand.

New issue concerning the design can be found here: https://github.com/comit-network/comit-i/issues/93

File: src/pages/SwapPage/Swap.tsx [hunk 2]
```diff
@@ -38,6 +58,67 @@ function Swap({ swap }: SwapProps) {
...
+        actionExecutionStatus,
+        activeLedgerActionDialog,
+        activeSirenParameterDialog
+      },
+      sideEffect
+    },
+    dispatch
+  ] = useReducer(reducer, initialState);
+
+  useEffect(() => {
```
Comment (thomaseizinger) [inline src/pages/SwapPage/Swap.tsx]: This is now duplicated right? Can we extract that into a custom hook? (i.e. just extract the function and name it something starting with `use`.

**Pull request merged**